internal/store/global_alerts.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
+	"github.com/mongodb/mongocli/internal/config"
+)
+
+type GlobalAlertLister interface {
+	GlobalAlerts(*om.AlertsListOptions) (*om.GlobalAlerts, error)
+}
+
+type GlobalAlertsStore interface {
+	GlobalAlertLister
+}
+
+// GlobalAlerts encapsulate the logic to manage different cloud providers
+func (s *Store) GlobalAlerts(opts *om.AlertsListOptions) (*om.GlobalAlerts, error) {
+	switch s.service {
+	case config.OpsManagerService, config.CloudManagerService:
```

Comment:
I think this is `config.OpsManagerService` only so you may need to remove `config.CloudManagerService`